core/src/main/java/org/mule/api/util/CredentialsMaskUtil.java

```diff
@@ -79,6 +80,19 @@ public static String maskUrlUserAndPassword(String input, Pattern passwordPatter
     }
 
     /**
+     * masks credentials in input prefixed
+     * 
+     * @param input input for user and password to be masked
+     * 
+     * @return input with user and password masked
+     */
+    public static String maskUrlCredentialsPrefixed(String input)
+    {
+        String inputMasked = maskUrlPattern(input, CREDENTIALS_PATTERN_PREFIX, PASSWORD_MASK);
```

Comment:
Is this mask utility useful for any URL or just for JDBC? JDBC connection strings are not real URLs... seems to me that this should be in the DB module